useful_resources/sample_code/s3_to_s3.py

```diff
@@ -1,26 +1,41 @@
+### METADATA
+
+# Connectors: S3
+# Description: Gets files from vendor s3 bucket and moves to own S3 bucket
+
+
+### CONFIGURATION
+
+# Set the configuration variables below or set environmental variables of the same name and leave these
+# with empty strings.  We recommend using environmental variables if possible.
+
+config_vars = {
+        # S3 (source) (note: this assumes a Civis Platform parameter called "AWS")
+        "S3_TEMP_BUCKET": "",
+        "AWS_ACCESS_KEY_ID": "",
+        "AWS_SECRET_ACCESS_KEY": "",
+        # S3 (vendor) (note: this assumes a Civis Platform parameter called "AWS_VENDOR")
+        'AWS_VENDOR_SECRET_ACCESS_KEY': "",
```

Comment:
If we repurpose as "source" / "destination", we would need to change these environment variable keys.. `AWS_SOURCE_SECRET_ACCESS_KEY`, `AWS_DESTINATION_SECRET_ACCESS_KEY`, etc.